options/linux/meson.build

```diff
@@ -87,6 +87,7 @@ if not no_headers
 	)
 	install_headers(
 		'include/linux/libc-compat.h',
+		'include/linux/input.h'
```

Comment:
While not needed (see other comments), you are missing a comma here, failing CI. Just a note for future PRs.